CHANGELOG.md

```diff
@@ -28,6 +28,28 @@
 
   ([PR #N](https://github.com/alphagov/digitalmarketplace-govuk-frontend/pull/N))
 
+🆕 New features:
+
+- New component: Digital Marketplace Alert component
+
+  *Requirement before using this component:*
+  - To use this the application must be using `govuk-frontend` template
+  - To use this the application must be using `govuk-frontend` phase banner as phase tag
+    is no longer part of the header and sits underneath and outside the header
```

Comment:
I don't think this is true. We should be able to use this without any other work.